### PR TITLE
55 style front home page closes #59 #55 #63 #62 #64 #65 #42 #70

### PR DESCRIPTION
**Pull request by bad6e**

### CODE REVIEW

File: db/migrate/20150925033708_add_column_to_candidates.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddColumnToCandidates < ActiveRecord::Migration
```
Comment (travishaby) [inline db/migrate/20150925033708_add_column_to_candidates.rb:1]: Cool, this is to connect the candidates to the main page right?

File: app/views/welcome/index.html.erb [hunk 2]
```diff
@@ -3,152 +3,159 @@
...
+
+				<!-- Candidate Section -->
+				<div class="row">
+						<div class="col-lg-12">
+								<h2 class="page-header" id="candidates"> Top Candidates</h2>
+						</div>
+						<% @featured_candidates.each do |candidate| %>
+						<div class="col-md-4 col-sm-6">
+								<a href="portfolio-item.html">
+										<%= link_to image_tag("#{candidate.last_name}.jpg", class:"img-responsive img-portfolio img-hover"), candidate_path(candidate.id)%>
```
Comment (travishaby) [inline app/views/welcome/index.html.erb]: This is where you change the picture for each candidate right?

**Comment by travishaby** [comment]
this is legit! once those tests are written lets merge this bad boy

File: db/migrate/20150925033708_add_column_to_candidates.rb (hunk 1 above)
Comment (bad6e) [inline db/migrate/20150925033708_add_column_to_candidates.rb:1]: Yeah - this was suppose to be "AddColumn'Featured'ToCandidates" but when I ran rails g migration I typed it in wrong. The code below it is correct

File: app/views/welcome/index.html.erb (hunk 2 above)
Comment (bad6e) [inline app/views/welcome/index.html.erb]: Correct - based on the new column I made "last_name". I am interpolating through each candidate and calling the associated .jpg. But the way we had it stored "Donald Trump".jpg would not work so I had to make a column I could reference to pull out the images.

**Pull request merged**